src/link/C/zig.h

```diff
@@ -750,3 +750,97 @@ static inline uint128_t zig_bit_reverse_u128(uint128_t value, uint8_t zig_type_b
 }
 
 #define zig_bit_reverse_i128 zig_bit_reverse_u128
+
+static inline float zig_div_truncf(float numerator, float denominator) {
+    return __builtin_truncf(numerator / denominator);
+}
+
+static inline double zig_div_trunc(double numerator, double denominator) {
+    return __builtin_trunc(numerator / denominator);
+}
+
+static inline long double zig_div_truncl(long double numerator, long double denominator) {
+    return __builtin_truncf(numerator / denominator);
+}
+
+#define zig_div_trunc_f16  zig_div_truncf
+#define zig_div_trunc_f32  zig_div_truncf
+#define zig_div_trunc_f64  zig_div_trunc
+#define zig_div_trunc_f80  zig_div_truncl
+#define zig_div_trunc_f128 zig_div_truncl
+
+#define zig_div_trunc_int(ZigType, CType) \
+    static inline CType zig_div_trunc_##ZigType(CType numerator, CType denominator) { \
+        return numerator / denominator; \
+    }
```

Comment:
If this is how div trunc for integers is defined then why don't we simply emit `a / b` instead of a call to this function? Looks like before this change, that's exactly what we did. Was there a problem with it?

Reply:
Very true, this was only convenient for a unified interface but we shouldn't need this code at all.

I changed the code in `genBody()` to instead selectively dispatch to `airBinOp(..., " / ")` or `airBinOpBuiltinCall(..., "div_trunc")` depending on the type, PTAL.